samza/indexer/src/main/java/org/sunbird/jobs/samza/service/Service.java

```diff
@@ -0,0 +1,111 @@
+package org.sunbird.jobs.samza.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.models.Constants;
+import org.sunbird.models.Message;
+import org.sunbird.models.MessageCreator;
+
+import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigValue;
+
+/**
+ * @author iostream04
+ *
+ */
+public class Service {
+  private static final String INDEX = ProjectUtil.EsIndex.sunbird.getIndexName();
+ 
+  private static MessageCreator messageCreator = null;
+  private final static String confFile = "indexMapping.conf";
+  private static com.typesafe.config.Config config = null;
+  private static Map<String, String> properties = null;
+
+  public Service() {
+    messageCreator = new MessageCreator();
+    config = ConfigFactory.parseResources(confFile);
+    initProps();
+    ProjectLogger.log("Service : intialized", LoggerEnum.INFO);
+  }
+
+  private void initProps() {
+    properties = new HashMap<>();
+    Set<Entry<String, ConfigValue>> confs = config.entrySet();
+    for (Entry<String, ConfigValue> conf : confs) {
+      properties.put(conf.getKey(), conf.getValue().unwrapped().toString());
+    }
+
+  }
+
+  public void process(Map<String, Object> messageMap) {
+    Message message = messageCreator.getMessage(messageMap);
+    Map<String, String> esTypesToUpdate = getEsTypeToUpdate(message);
```

Comment:
Perform some validations on the input message as you are expecting objectType, operation etc.